bower.json

```diff
@@ -32,5 +32,8 @@
     "angular-ckeditor": "~1.0.0",
     "roboto-condensed": "~0.3.0",
     "open-sans-fontface": "https://github.com/OpenSlides/open-sans.git#1.4.2.post1"
+  },
+  "resolutions": {
+    "angular": "^1.2.x"
```

Comment:
That will be override by #1903. angular-pdf needs angular 1.4.8+. So merge this here.